### DISCUSSION TITLE
I want to add the horizontal scroll bar on the top also

### DISCUSSION THREAD

**Discussion by sant527-sms-magic** [KevinVandy/material-react-table#596] [Q&A]
```
material-react-table: 1.12.1
 "react": "17.0.2",
"@mui/material": "5.4.1",
```

and my table is 
```
        <MaterialReactTable
          columns={stateTable.recentSmsColumnsRed}
          data={stateTable.recentSmsDataRed}
          enablePagination={false}
          initialState={{
            density: "compact",
          }}
          enableRowNumbers
          rowNumberMode="original"
          enableStickyHeader
        />
```

I have lot of columns and i want to have a horizontal scroll bar on the top also.

**Comment by KevinVandy** [comment]
Is that possible with CSS or something? I've not really ever seen that before

**Comment by sant527-sms-magic** [comment]
Hi

What's the solution

**Reply by KevinVandy** [reply]
There is none unless you know how

**Reply by sant527-sms-magic** [reply]
![image](https://github.com/KevinVandy/material-react-table/assets/129738367/0276aec6-6417-4da2-b42b-4ddb3fdde6b6)
I want to know again, if this is possible

currently its only in the bottom
![image](https://github.com/KevinVandy/material-react-table/assets/129738367/dfe2e80e-a154-4502-aac1-2c3573882317)
